.cicd/scripts/srw_ftest.sh

```diff
@@ -95,8 +92,10 @@ module load wflow_${platform,,}
 conda activate srw_app
 set -e -u
 
-# Adjust for strict limitation of stack size 
-sed "s|ulimit -s unlimited;|ulimit -S -s unlimited;|" -i ${workspace}/ush/machine/hera.yaml
+echo "${user_settings}" | uw config realize \
+  -i config.community.yaml \
+  -o config.yaml \
+  --update_format yaml
```

Comment:
For -i, the full path needs to be included.  This is also necessary for the -o option.  Finally, --update_format should be --update-format.

```suggestion
  -i ${workspace}/ush/config.community.yaml \
  -o ${workspace}/ush/config.yaml \
  --update-format yaml
```